### PR TITLE
[MINOR][DOC] Update header links and image files

### PR DESCRIPTION
**Pull request by j143**
**What this PR do?**

- Update header links.
- Add convolution image files

**TL;DR:**

image

### CODE REVIEW

**Comment by Baunsgaard** [comment]
Hi Janathan, this does not work once you go into any sub-page,
since each of the links refer to relative paths from where you are.

This means that from the main page it links correctly, from lets say DML Language Reference it does not.

File: docs/_includes/header.html [hunk 1]
```diff
@@ -35,10 +35,37 @@
...
+                        <li><b>Tools:</b></li>
+                        <li><a href="site/debugger-guide.html">Debugger Guide</a></li>
+                        <li><a href="site/developer-tools-systemds.html">IDE Guide</a></li>-->
+                        <li class="divider"></li>
+                        <li><b>Other:</b></li>
+                        <li><a href="https://github.com/apache/systemds/blob/master/CONTRIBUTING.md">Contributing to SystemDS 🡕</a></li>
+                        <!--
+                        <li><a href="site/engine-dev-guide.html">Engine Developer Guide</a></li>
+                        <li><a href="site/troubleshooting-guide.html">Troubleshooting Guide</a></li>
+                        <li><a href="site/release-process.html">Release Process</a></li> -->
```
Comment (Baunsgaard) [inline docs/_includes/header.html:63]: Also I would remove these uncommented links

Comment (j143) [reply docs/_includes/header.html:63]: Happy to remove these.

These are the document pages necessary to be provided in the future. 😃

**Comment by j143** [comment]
> this does not work once you go into any sub-page,
> since each of the links refer to relative paths from where you are.

Is it possible to correct the links based on how you intend the website to deploy. Can you do that? Thanks.

cc @Baunsgaard

**Comment by Baunsgaard** [comment]
thanks for the PR i have merged it now. I did some minor modifications such as adding an option to go back to the main overview page..

**Pull request closed without merging**